frontend/src/modules/resource-not-found/resource-not-found.tsx

```diff
@@ -22,7 +22,7 @@ import {
     setBreadcrumbs,
     setItemsForProjectName,
 } from '../../shared/layout/navigation/navigation.reducer';
-import { useAppDispatch } from '../../config/store';
+import {useAppDispatch} from '../../config/store';
```

Comment:
I think this was changed on accident, but I won't block if it somehow sneaks by eslint. Seeing this in several imports. I thought that eslint did check for this though.